### DISCUSSION TITLE
local supabase storage limit

### DISCUSSION THREAD

**Discussion by MMTE** [supabase/supabase#21903] [Questions] [answered]
I'm going to use self-hosted version of supabase. Although I have the local version and it's not hosted online, the maximum file size is 50MB which is maybe good for free online hosted free plan but it would not enough for self hosted version in production. 
I would appreciate it if anybody could help whats wrong with this situation?

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/self-hosting/storage/config has a file size limit setting.

**Reply by MMTE** [reply]
That's write but I don't know why I had a unhealthy database running and that was a mess.
finally worked. thx
